### PR TITLE
Configure monitoring with Prometheus

### PR DESCRIPTION
**Pull request by martin-helmich**

### CODE REVIEW

File: _modules/mwdocker.py [hunk 1]
```diff
@@ -90,11 +90,12 @@ def image_id(image):
     return None
 
 
-def delete_container(name):
+def delete_container(name, timeout=10, with_volumes=False):
```
Comment (jkmw) [inline _modules/mwdocker.py:93]: missing comment for timeout param

File: _states/mwdocker.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 
 def running(name, image, volumes=(), restart=True, tcp_ports=(), udp_ports=(), environment=None, command=None, dns=None,
-            domain=None, volumes_from=None, links=None, user=None, warmup_wait=60, stateful=False):
+            domain=None, volumes_from=None, links=None, user=None, warmup_wait=60, stateful=False, labels=None):
```
Comment (jkmw) [inline _states/mwdocker.py:24]: missing comment for timeout param

Comment (martin-helmich) [inline _states/mwdocker.py:24]: Fixed

File: _modules/mwdocker.py (hunk 1 above)
Comment (martin-helmich) [inline _modules/mwdocker.py:93]: Fixed

File: mwms/monitoring/cadvisor.sls [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+/usr/local/sbin/cadvisor:
+  file.managed:
+    - source: salt://mwms/monitoring/files/cadvisor
+    - mode: 755
+
+/etc/supervisor/conf.d/cadvisor.conf:
+  file.managed:
+    - contents: |
+        [program:cadvisor]
+        command=/usr/local/sbin/cadvisor -port {{ cadvisor_port }}
```
Comment (jkmw) [inline mwms/monitoring/cadvisor.sls]: perhaps put the cadvisor params into a pillar

File: mwms/monitoring/cadvisor.sls [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    - watch_in:
+      - service: supervisor
+
+/etc/consul/service-cadvisor.json:
+  file.managed:
+    - contents: |
+        {
+          "service": {
+            "name": "cadvisor",
+            "port": {{ cadvisor_port }}
```
Comment (jkmw) [inline mwms/monitoring/cadvisor.sls]: perhaps put the cadvisor params into a pillar

File: mwms/monitoring/cadvisor.sls (hunk 3 above)
Comment (martin-helmich) [inline mwms/monitoring/cadvisor.sls]: done

Comment (jkmw) [inline mwms/monitoring/cadvisor.sls]: I meant to put the whole arguments for cadvisor into a pillar.

For example:
`command=/usr/local/sbin/cadvisor {{ cadvisor_arguments | join(',') }}`

**Pull request merged**